### PR TITLE
CSS tweaks

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
- Added hover/focus styles to colorpicker button
- Repositioned arrow inside button

### CODE REVIEW

File: src/app/public/testing/colorpicker-fixture.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class SkyColorpickerFixture {
   public async setValueFromPresets(presetIndex: number): Promise<any> {
     await this.clickColorpickerButtonEl();
 
-    const presetColors = document.querySelectorAll('.sky-preset-color');
+    const presetColors = document.querySelectorAll('.sky-colorpicker-preset-color-area button');
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/testing/colorpicker-fixture.ts:92]: Not a part of the scope of this PR, but the test fixture spec needed to be updated to be more flexible now that we have modern theme enabled.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-css-fixes](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-css-fixes)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Review by Blackbaud-AdamFunderburk — DISMISSED** [review]

File: src/app/public/modules/colorpicker/colorpicker.component.scss [hunk 2]
```diff
@@ -286,14 +286,26 @@
...
   }
 
+  .sky-colorpicker-button:focus {
+    box-shadow: inset 0 0 0 2px #1870B8, 0px 1px 8px 0px rgba(0, 0, 0, 0.3);
+    outline: none;
+    text-decoration: none;
+  }
+
+  .sky-colorpicker-button:hover {
+    box-shadow: inset 0 0 0 1px #1870B8;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/colorpicker/colorpicker.component.scss:296]: Is this hex value represented as a SCSS variable? (Same as above.)

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/colorpicker/colorpicker.component.scss]: Yep. Just had to find it. Added.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-css-fixes](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-css-fixes)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**